Source/CesiumRuntime/Private/CesiumGlobeAnchorComponent.cpp

```diff
@@ -411,15 +411,18 @@ void UCesiumGlobeAnchorComponent::SetEastSouthUpRotation(
           VecMath::createQuaternion(EastSouthUpRotation),
           scale);
 
-  const CesiumGeospatial::Ellipsoid& ellipsoid =
-      this->ResolveGeoreference()->GetEllipsoid()->GetNativeEllipsoid();
-
-  anchor.setAnchorToLocalTransform(
-      eastSouthUp,
-      newModelToEastSouthUp,
-      false,
-      ellipsoid);
-  this->_updateFromNativeGlobeAnchor(anchor);
+  const ACesiumGeoreference* georeference = this->ResolveGeoreference();
+  if (georeference) {
```

Comment:
Prefix pointer variables with `p` in accordance with our [style guide](https://cesium.com/learn/cesium-unreal/ref-doc/style-guide.html#public-vs-private-style).

```suggestion
  const ACesiumGeoreference* pGeoreference = this->ResolveGeoreference();
  if (pGeoreference) {
```